modules/SofaMiscFem/src/SofaMiscFem/TriangularFEMForceField.inl

```diff
@@ -282,19 +290,19 @@ void TriangularFEMForceField<DataTypes>::reinit()
         setYoungArray(f_young.getValue());
 
 #ifdef PLOT_CURVE
```

Comment:
I know this is the not the topic of the PR but this PLOT_CURVE looks sucha broken design.

Reply:
It will be removed in next PR when TriangleFEM and TriangularFEM will be unify

Reply:
I swear ;)